src/main/java/gregtech/loaders/recipe/MiscRecipeLoader.java

```diff
@@ -28,6 +28,11 @@
 
     public static void init() {
 
+        // Basic Terminal Recipe
+        ModHandler.addShapedRecipe(true, "basic_terminal", TERMINAL.getStackForm(),
+                "SPS", "PBP", "PWP", 'S', new UnificationEntry(screw, WroughtIron), 'P', OreDictUnifier.get("paneGlass"), 'B', new ItemStack(Items.BOOK),
+                                        'P', new UnificationEntry(plate, WroughtIron), 'W', new UnificationEntry(cableGtSingle, RedAlloy));
```

Comment:
Since this recipe is for making the terminal in case the player loses the one that they are given on log in, I am ok with it being a little more expensive and requiring Red Alloy. However, this was supposed to be a wire instead of a cable, so you don't have to go to the nether to get sulfur at all (which would require steel to light the nether portal)